src/pages/en/guides/imports.mdx

```diff
@@ -79,16 +79,17 @@ While Astro understands JSX syntax out-of-the-box, you will need to include a fr
 
 ### NPM Packages
 
+If you've installed an NPM package, you can import it in Astro.
+
 ```astro
 ---
 import { Icon } from 'astro-icon';
 ---
 ```
-If you've installed an NPM package, you can import it in Astro. Even if a package was published using a legacy format, Astro will convert the package to ESM so that `import` statements work.
+If a package was published using a legacy format, Astro will try to convert the package to ESM so that `import` statements work. In some cases, you may need to adjust your [`vite` config](/en/reference/configuration-reference/#vite) for it to work.
 
-:::warning
 Some packages rely on a browser environment. Astro components runs on the server, so importing these packages in the frontmatter may [lead to errors](/en/guides/troubleshooting/#document-or-window-is-not-defined).
-:::
+
```

Comment:
I noticed that this `:::warning` admonition thing was not doing anything in the live version so i removed it.

Reply:
Ah, this is because it should have been `:::caution` — mind adding it back? Might as well fix it while we’re here.

Reply:
done

Reply:
Wow! Lightspeed dev 🚀